### PR TITLE
Create initial boilerplate for additional TTY numbers

### PR DESCRIPTION
**Pull request by kelsonic**
## Description
**Issue:** https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/pull/341

This PR adds some additional TTY numbers.

## Testing done
N/A

## Screenshots
N/A

## Acceptance criteria
- [x] Add additional TTY numbers to Telephone

## Definition of done
- [x] Changes have been tested in vets-website
- [x] Changes have been tested in IE11, if applicable
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by kelsonic** [comment]
This PR is awaiting feedback from @DanielleThierryUSDSVA and @RLHecht for TTY number labels for the following numbers:

```
800-877-3399
800-877-8339
800-799-4889
866-363-2883
711
```

An example can be found [here](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/visual-design/components/telephone/) for the other `<Telephone />` numbers:

image

**Comment by kelsonic** [comment]
`800-877-3399` - May be a typo?

`800-877-8339` - Federal Relay Service @RLHecht Can you look into this one as well as a double check? It's also showing search results for HUD and other mortgage services specifically.

`800-799-4889` - Suicide Prevention Lifeline

`866-363-2883` - Defense Manpower Data Center (DMDC)/Defense Enrollment Eligibility Reporting System (DEERS) Support Office

`711` - Telecommunications Relay Service

**Comment by Mottie** [comment]
> `800-877-3399` - May be a typo?

If it's a typo, it's in the vets-website repo in 13 places.

**Comment by kelsonic** [comment]
Ya, @DanielleThierryUSDSVA isn't sure what that number is so was just a hypothesis. That said, if it's in `vets-website` 13 times the probability of it being a typo is pretty low I'd think...

**Comment by Mottie** [comment]
Haha, I called that number... it's like a sales site or something. I press `0` at one point instead of the number they wanted and it hung up on me. I think it's safe to say that isn't a TTY number.

File: packages/formation-react/src/components/Telephone/Telephone.mdx [hunk 1]
```diff
@@ -126,18 +126,22 @@ import Telephone, { CONTACTS, PATTERNS } from '@department-of-veterans-affairs/f
...
 | `DS_LOGON_TTY` | `8663632883` | Defense Manpower Data Center TTY |
+| `DS_LOGON` | `8005389552` | Defense Manpower Data Center |
+| `FEDERAL_RELAY_SERVICE` | `8008778339` | Federal Relay Service |
 | `GI_BILL` | `8884424551` | Education Call Center (1-888-GI-BILL-1) |
 | `GO_DIRECT` | `8003331795` | Go Direct/Direct Express (Treasury) |
 | `HELP_DESK` | `8555747286` | VA Help desk |
 | `HELP_TTY` | `8008778339` | VA Help Desk TTY |
 | `MY_HEALTHEVET` | `8773270022` | My HealtheVet help desk |
 | `NCA` | `8005351117` | National Cemetery Scheduling Office |
+| `SUICIDE_PREVENTION_LIFELINE` | `8007994889` | Suicide Prevention Line |
```
Comment (Mottie) [inline packages/formation-react/src/components/Telephone/Telephone.mdx:144]: Should we preemptively add `988`?

File: packages/formation-react/src/components/Telephone/Telephone.mdx [hunk 2]
```diff
@@ -126,18 +126,22 @@ import Telephone, { CONTACTS, PATTERNS } from '@department-of-veterans-affairs/f
 |:----|:-------|:------------|
 | `222_VETS` | `8772228387` | VA Help Line |
 | `4AID_VET` | `8774243838` | National Call Center for Homeless Veterans |
+| `711` | `711` | Telecommunications Relay Service |
 | `911` | `911` | Emergency! |
 | `CAREGIVER` | `8552603274` | VA National Caregiver Support Line |
 | `CRISIS_LINE` | `8002738255` | Veterans Crisis hotline |
 | `CRISIS_TTY` | `8007994889` | Veterans Crisis hotline TTY |
```
Comment (Mottie) [inline packages/formation-react/src/components/Telephone/Telephone.mdx:133]: Should we also add a TTY pattern? `TTY: ###-###-####`?

File: packages/formation-react/src/components/Telephone/Telephone.mdx (hunk 1 above)
Comment (kelsonic) [reply packages/formation-react/src/components/Telephone/Telephone.mdx:144]: Hmm, you cool if we wait until we actually get a use-case for it on vets-website? Just want to make sure we don't add unused constants

**Comment by kelsonic** [comment]
@department-of-veterans-affairs/frontend-review-group ready for review! 🎉

**Pull request merged**